File: 2014-10-CSharp-Fundamentals-1/5. Conditional Statements/IfElseStatement/IfElseStatement.cs
using System;

class IfElseStatement
{
    static void Main()
    {
        Console.Write("Please enter a number: ");
        int number = int.Parse(Console.ReadLine());

        if (number % 2 == 0)
        {
            Console.WriteLine("This number is even.");
        }
        else
        {
            Console.WriteLine("This number is odd.");
        }
    }
}

File: 2015-10-Java-Fundamentals-1/05. Conditional-statements/demos/IfElseStatement.java
import java.util.Scanner;

public class IfElseStatement {
    public static void main(String[] args) {
        Write("Please enter a number: ");

        Scanner scanner = new Scanner(System.in);
        int number = scanner.nextInt();

        if (number % 2 == 0) {
            writeLine("This number is even.");
        }
        else {
            writeLine("This number is odd.");
        }
    }

    public static void Write(Object obj) {
        System.out.print(obj);
    }

    public static void writeLine(Object obj) {
        System.out.println(obj);
    }
}